boards/arm/nucleo_g474re/nucleo_g474re_defconfig

```diff
@@ -36,11 +35,11 @@ CONFIG_CLOCK_STM32_PLL_Q_DIVISOR=2
 CONFIG_CLOCK_STM32_PLL_R_DIVISOR=2
 
 # Produce 150MHz clock at PLLCLK output
-CONFIG_CLOCK_STM32_PLL_N_MULTIPLIER=75
+# CONFIG_CLOCK_STM32_PLL_N_MULTIPLIER=75
 
-# Comment out above and uncomment below for 170MHz. Note that you
-# must have configured the mcu for boost power mode.
-# CONFIG_CLOCK_STM32_PLL_N_MULTIPLIER=85
+# Comment out above and uncomment below for 170MHz.
```

Comment:
This is no longer true.

Reply:
proposal : 

> Comment out below and uncomment above for less than 170MHz.

Reply:
ok, but then the note below does not apply anymore.